app-vim/bufexplorer: bump EAPI 6 -> 8, add 7.4.26

laumann opened the pull request:
Bug: https://bugs.gentoo.org/830866

laumann commented:
...

10 months later... Two patch versions were released, so I've rebased on master ::gentoo and updated the new version.

@a17r I kinda doubt @gentoo/vim will act here, but let's see :-)

@mattst88 @jsmolic the git logs tell me you are the latest committers on this package, maybe one of you will approve/merge?

mattst88 commented:
Unfortunately it looks like you'll have to rebase again.

I just dropped some old versions. I don't know anything about this package. Sorry.

laumann commented:
> Unfortunately it looks like you'll have to rebase again.

I'll just close this, 7.4.25 got added the very same day I updated this branch, so I won't get in the way. Apparently I had missed all the vim ebuild updates in #30077.

> I just dropped some old versions. I don't know anything about this package. Sorry.

No worries, thanks for the input.
